engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/keys/ScatterCollectorKey.scala

```diff
@@ -15,4 +17,19 @@ private [execution] case class ScatterCollectorKey(node: GraphNode,
   override val index = None
   override val attempt = 1
   override val tag = s"Collector-${node.localName}"
+
+  def processRunnable(data: WorkflowExecutionActorData): ErrorOr[WorkflowExecutionDiff] = {
+    data.valueStore.collectShards(this) map { outputs =>
+      // TODO WOM: fix
```

Comment:
create issue perhaps?

Reply:
Yeah this is related to conditionals so I'd say the "Reinstate conditional" ticket covers it but I'll link to it specifically in the TODO.